src/common/types/metadataQueries.js

```diff
@@ -16,6 +16,14 @@ type MetadataQueryOrderByClause = {
     field_key: string,
 };
 
+type MetadataFieldConfig = {
+    canEdit?: boolean,
+    displayName?: string,
+    key: string,
+};
+
+type MetadataFieldsToShow = Array<MetadataFieldConfig | string>;
```

Comment:
I'm guessing that the two types are from the API, but it's not immediately clear to me why a metadata field might be a string. Can you add a comment in the code with an explanation of this case?

Reply:
`MetadataFieldsToShow` type is not from API. This is a prop that developers can pass to the <ContentExplorer />. It could be simple as string or it could be an config object
```
const mdFieldsToShow1: MetadataFieldsToShow = ['name', 'desc'];
const mdFieldsToShow2: MetadataFieldsToShow = [
    { key: 'name', displayName: 'Item Name', canEdit: false },
    { key: 'desc', displayName: 'Description', canEdit: true },
];
```

I will add a comment.